geometry_msgs/Point point
float64[3] direction_vector
